core/src/main/scala/org/broadinstitute/dsde/rawls/rbs/RbsService.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.rawls.rbs
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import bio.terra.rbs.generated.model.PoolInfo
+import com.google.api.client.auth.oauth2.Credential
+import org.broadinstitute.dsde.rawls.dataaccess.rbs.RbsDAO
+import org.broadinstitute.dsde.rawls.model.{GoogleProjectId, UserInfo}
+
+import scala.concurrent.ExecutionContext
+
+object RbsService {
+  def constructor(rbsDao: RbsDAO, serviceAccountCreds: Credential)(userInfo: UserInfo)(implicit executionContext: ExecutionContext): RbsService = {
+    new RbsService(rbsDao, userInfo, serviceAccountCreds)
+  }
+}
+// todo: delete this file? might not need this if we're just using RBS under the hood.
```

Comment:
I think it depends on how much work needs to get done here beyond just making calls to RBS. `RbsDAO` should just be the API calls and some retry logic like you have it in my opinion. In its current state, I'd agree that we could probably use `RbsDAO` directly from `WorkspaceService`, but I could see wanting to move stuff like `generateHandoutRequestId`, `getProjectPoolId` and `getGoogleProjectFromRBS` into this class